forc-pkg/src/pkg.rs

```diff
@@ -1461,9 +1464,10 @@ fn fetch_deps(
         })?;
 
```

Comment:
Could we change the error message on line 1419 from `Failed to source dependency` to `Failed to source dependency {dependency name}` ?

Reply:
#4685